src/lib/storage/vector_compression/bitpacking/bitpacking_vector_type.hpp

```diff
@@ -0,0 +1,11 @@
+#pragma once
+
+#include "compact_vector.hpp"
+#include "types.hpp"
+
+namespace opossum {
+
+template <typename T>
+using pmr_compact_vector = compact::vector<T, 0u, uint64_t, PolymorphicAllocator<uint64_t>>;
```

Comment:
Will later add a comment to whole uint32_t issue. If we don't reuse the this name for anything than the vector compression, we can also use uint32_t here.

Reply:
Please add a comment why (for now) uint32_t is sufficient.